subsys/bluetooth/host/iso.c

```diff
@@ -729,6 +724,61 @@ void bt_iso_recv(struct bt_conn *iso, struct net_buf *buf, uint8_t flags)
 }
 #endif /* CONFIG_BT_ISO_RX */
 
+static bool iso_has_data(struct bt_conn *conn)
+{
+#if defined(CONFIG_BT_ISO_TX)
+	return !k_fifo_is_empty(&conn->iso.txq);
+#else
+	return false;
+#endif
```

Comment:
General comment: It's nice to add the conditional when we add guards like this, so that it's easier to determine the cases:

```suggestion
#if defined(CONFIG_BT_ISO_TX)
	return !k_fifo_is_empty(&conn->iso.txq);
#else /* !CONFIG_BT_ISO_TX */
	return false;
#endif /* CONFIG_BT_ISO_TX */
```

Reply:
my opinion is that if the `#ifdef` is two or less lines above, the comment adds useless visual clutter. 
I'll apply the suggestion anyways, but that's just like my opinion man. Another entry for https://github.com/zephyrproject-rtos/zephyr/issues/55281

Reply:
We can avoid the problem by using `COND_CODE_1`.

Reply:
`IF_ENABLED` is also an option

Reply:
both have a somewhat lispy syntax which is alien to non-zephyr C developers. I'd rather avoid them if I can.